src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -1,22 +1,28 @@
 package com.minecolonies.colony;
 
 import com.minecolonies.MineColonies;
-import com.minecolonies.colony.buildings.AbstractBuilding;
-import com.minecolonies.colony.buildings.BuildingHome;
-import com.minecolonies.colony.buildings.BuildingTownHall;
+import com.minecolonies.achievements.AbstractSizeAchievement;
+import com.minecolonies.achievements.AchSizeCity;
+import com.minecolonies.achievements.AchSizeSettlement;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.achievements.AchSizeSettlement'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)